mirage/seed_image/catalog_seed_image.py

```diff
@@ -3390,7 +3403,7 @@ def make_extended_source_image(self, extSources, extStamps):
         for entry, stamp in zip(extSources, extStamps):
             stamp_dims = stamp.shape
 
-            stamp *= entry['counts_per_frame_e']
+            stamp *= entry['countrate_e/s']
```

Comment:
Is this an separate unrelated bug that was fixed as part of this PR? Seems unrelated to PSF normalization.

Reply:
Yes, this is a separate bug that I had fixed in a different branch but needed here as well in order to test for the correct source brightnesses.